NadekoBot.Core/Modules/Utility/QuoteSlashCommands.cs

```diff
@@ -335,7 +336,19 @@ public async Task QuoteAddFile([Summary("keyword","Name of quote")]string keywor
                 }
             
                 keyword = keyword.ToUpperInvariant();
+                byte[] filebytes;
                 string text = file.Url;
+                if (file.Ephemeral){
+                    using (var http = _httpFactory.CreateClient()) {
```

Comment:
nit: remove the space between the last closing bracket and opening curly brace. It's cool to stay consistent
![image](https://user-images.githubusercontent.com/32067562/160511988-dfca40cc-88f4-4d5f-b66f-12976127b665.png)